CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 * Basic support for editing tables by adding `table` to the rich text toolbar. Enabling `table` allows you to create tables, including `td` and `th` tags, with the ability to merge and split cells. For now the table editing UI is basic, all of the functionality is there but we plan to add more conveniences for easy table editing soon. See the "Table" dropdown for actions that are permitted based on the current selection.
 * Support for `div` tags in the rich text toolbar, if you choose to include them in `styles`. This is often necessary for A2 content migration and can potentially be useful in new work when combined with a `class` if there is no suitable semantic block tag.
 * The new `@apostrophecms/attachment:download-all --to=folder` command line task is useful to download all of your attachments from an uploadfs backend other than local storage, especially if you do not have a more powerful "sync" utility for that particular backend.
+* For the widget `image-widget`
```

Comment:
Can you combine the first two lines into a single sentence, please? Something like, "Two new options have been added to the `image-widget`: `loadingType` and `size`."

Reply:
Is the wording now better?

Reply:
Yup, looks good to me.